plugins/catalog-backend/src/database/DatabaseManager.ts

```diff
@@ -18,17 +18,36 @@ import Knex from 'knex';
 import path from 'path';
 import { Logger } from 'winston';
 import { CommonDatabase } from './CommonDatabase';
-import type { Database } from './types';
+import { Database } from './types';
 
 export class DatabaseManager {
   public static async createDatabase(
-    knex: Knex,
+    database: Knex,
     logger: Logger,
   ): Promise<Database> {
-    await knex.migrate.latest({
+    await database.migrate.latest({
       directory: path.resolve(__dirname, 'migrations'),
       loadExtensions: ['.js'],
     });
-    return new CommonDatabase(knex, logger);
+    return new CommonDatabase(database, logger);
```

Comment:
hm i think actually all of the above lines are a regression because you weren't on latest master? i liked to call these knex instead of database because it was so confusing to send a database to a database etc :)

Reply:
Done